packages/python/plotly/requires-express.txt

```diff
@@ -0,0 +1,8 @@
+### Required dependencies for Plotly Express    ###
+###                                             ###
+### To install, run:                            ###
+###      $ pip install -r requires-express.txt  ###
+###                                             ###
+###################################################
+
+numpy
```

Comment:
We should add a version pin here, we want to recommend a specific version of Numpy

Reply:
I believe we shouldn't pin specific versions unless absolutely necessary — this is why:

https://iscinumpy.dev/post/bound-version-constraints/